llms/hf_llm/generate.py

```diff
@@ -0,0 +1,82 @@
+# Copyright © 2023 Apple Inc.
+
+import argparse
+import time
+
+import mlx.core as mx
+import models
+import transformers
+
+
+def generate(
+    model: models.Model,
+    tokenizer: transformers.AutoTokenizer,
+    prompt: str,
+    max_tokens: int,
+    temp: float = 0.0,
+):
+    prompt = tokenizer(args.prompt, return_tensors="np", return_attention_mask=False,)[
```

Comment:
Completely nitpicky but that probably doesn't pass black cause of the last comma that remained when moving it from multi-line to single line.

Reply:
Huh that is weird. I ran `pre-commit run --all-files` and it didn't catch that... you are right though black doesn't like it.

Reply:
Ok I'm bumping the pre-commit black version in our config. It properly catches this now.